packages/data-stores/src/site/actions.ts

```diff
@@ -532,7 +531,9 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		);
 
 		yield runThemeSetupOnSite( siteSlug, design, {
-			trimContent: false,
+			// trimContent true ensures that the starter content is trimmed in case sourceSiteId is defined
+			// For instance only a max of three posts will be added to the user site
+			trimContent: true,
```

Comment:
The [setupSiteAfterCreation](https://github.com/Automattic/wp-calypso/blob/8caec7413604b7007a25f622e2af84e1f27d1f7f/packages/onboarding/src/setup-tailored-site-after-creation.ts#L74) also uses this action, do we want to enable the trimContent for the flow from there as well?

Reply:
As I understand, that condition only sets the intent. Can you confirm that?

[applyThemeWithPatterns](https://github.com/Automattic/wp-calypso/blob/8caec7413604b7007a25f622e2af84e1f27d1f7f/packages/onboarding/src/setup-tailored-site-after-creation.ts#L72) is called only when the theme is virtual. 

I'm assuming that any virtual theme, in any flow, will benefit from using `trimContent` because it has been always enabled until I suggested disabling it as a workaround to copy 4 posts, rather than 3, from BC3 demo site when the assembler still didn't use the site-assembler endpoint.

I see `trimContent` as a protection to prevent using big headstart annotations to avoid performance issues. This param limits the number of initial posts, pages, and menus and skips the images in posts. For reference, see the function **trim_starter_content** in fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Sahk%2Qurycref%2Sgevz%2Qnaabgngvbaf.cuc%3Se%3Qn467325s%2318-og